content/posts/hiero_links_of_may_eleven.md

```
@@ -0,0 +1,28 @@
+This week featured a lot of new and exciting updates within the Hiero Organization. 
+Below are links to some of things that happened this past week. **Check it out!**
+
+---
+
+**Angelina Ceppaluni** implemented [HIP 1261:Simple Fees](https://github.com/hiero-ledger/hiero-sdk-python/pull/2231) to support fee estimation in the Python SDK.
+
+**Achintya Sinha** corrected a windows [debug build failure](https://github.com/hiero-ledger/hiero-sdk-cpp/pull/1623) in FeeEstimateQuery.cc by correcting the include order to prevent MSVC macro collisions in the C++ SDK. 
+
+**Xin Li** implemented [TSS system transaction types](https://github.com/hiero-ledger/hiero-mirror-node/pull/13525) to REST to ensure TSS system transaction types are properly recognized in the REST transactions endpoint instead of appearing as unknown transaction types in Hiero Mirror Node.
+
+**Samrajniii** added a [broken link checker](https://github.com/hiero-ledger/hiero-sdk-java/pull/2729) script to check for broken links in the Java SDK.
```

markdownlint-cli2 (0.22.1)

[warning] Heading style
Expected: atx; Actual: setext

(MD003, heading-style)